docs/administration/Configuration.md

```diff
@@ -42,462 +42,2042 @@ ADMIN SET FRONTEND CONFIG ("key" = "value");
+
+#### load_data_reserve_hours
+
+- **Default**: 4
+- **Unit**: Hour
+- **Description**: The reservation time for the files produced by small-scale loadings.
+
+#### number_tablet_writer_threads
+
+- **Default**: 16
+- **Unit**: N/A
+- **Description**: The number of threads used for Stream Load.
+
+#### streaming_load_rpc_max_alive_time_sec
+
+- **Default**: 1200
+- **Unit**: Second
+- **Description**: The RPC timeout for Stream Load.
+
+#### fragment_pool_thread_num_min
+
+- **Default**: 64
+- **Unit**: N/A
+- **Description**: The minimum number of threads used for query.
+
+#### fragment_pool_thread_num_max
+
+- **Default**: 4096
+- **Unit**: N/A
+- **Description**: The maximum number of threads used for query.
+
+#### fragment_pool_queue_size
+
+- **Default**: 2048
+- **Unit**: N/A
+- **Description**: The upper limit of the query number that can be processed on each BE node.
+
+#### enable_token_check
+
+- **Default**: TRUE
+- **Unit**: N/A
+- **Description**: A boolean value to control whether to enable the token check. TRUE indicates enabling the token check, and FALSE indicates disabling it.
+
+#### enable_prefetch
+
+- **Default**: TRUE
+- **Unit**: N/A
+- **Description**: A boolean value to control whether to enable the pre-fetch of the query. TRUE indicates enabling pre-fetch, and FALSE indicates disabling it.
+
+#### load_process_max_memory_limit_bytes
+
+- **Default**: 107374182400
+- **Unit**: Byte
+- **Description**: The maximum size limit of memory resources that can be taken up by all load processes on a BE node.
+
+#### load_process_max_memory_limit_percent
+
+- **Default**: 30
+- **Unit**: %
+- **Description**: The maximum percentage limit of memory resources that can be taken up by all load processes on a BE node.
+
+#### sync_tablet_meta
+
+- **Default**: FALSE
+- **Unit**: N/A
+- **Description**: A boolean value to control whether to enable the synchronization of the tablet metadata. TRUE indicates enabling synchronization, and FALSE indicates disabling it.
+
+#### routine_load_thread_pool_size
+
+- **Default**: 10
+- **Unit**: N/A
+- **Description**: The thread pool size for Routine Load on each BE. Since v3.1.0, this parameter is deprecated. The thread pool size for Routine Load on each BE is now controlled by the FE dynamic parameter max_routine_load_task_num_per_be.
+
+#### brpc_max_body_size
+
+- **Default**: 2147483648
+- **Unit**: Byte
+- **Description**: The maximum body size of a BRPC.
+
+#### tablet_map_shard_size
+
+- **Default**: 32
+- **Unit**: N/A
+- **Description**: The tablet map shard size. The value must be a power of two.
+
+#### enable_bitmap_union_disk_format_with_set
+
+- **Default**: FALSE
+- **Unit**: N/A
+- **Description**: A boolean value to control whether to enable the new storage format of the BITMAP type, which can improve the performance of bitmap_union. TRUE indicates enabling the new storage format, and FALSE indicates disabling it.
```

Comment:
1. Remove "A boolean value to control", and make the sentence start with "Whether".
2. Wrap the parameter values "TRUE" and "FALSE" in code format.